styles.css

```diff
@@ -58,18 +63,207 @@
   align-items: center;
 }
 
-.connect > ul {
+.connect-ul {
   list-style: none;
   display: flex;
   margin-right: 220px;
 }
 
-.connect > ul > li {
+.connect-ul > li {
   padding: 6px;
   margin-top: -16px;
 }
 
-.connect > ul > li,
+.connect-ul > li,
 .hamburgerIcon > img {
   cursor: pointer;
 }
+
+.works-section {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  gap: 88px;
+  margin-top: 114px;
+}
+
+.work {
+  width: 327px;
+  height: 578px;
+  margin-left: 24px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+}
+
+.work > h3 {
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.work > ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  gap: 12px;
+  list-style: none;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.skill-info:not(:first-child) {
+  color: #7a869a;
+}
+
+.work > p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.work-lang {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+}
+
+.work-lang > li {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.work > button {
+  background: none;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 18px;
+  cursor: pointer;
+}
+
+.work > button:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.workHeight {
+  height: 622px;
+}
+
+.about-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 28px;
+}
+
+.about-section article {
+  background-color: white;
+  width: 375px;
+  margin-top: 96px;
+  padding: 24px;
+  border-top-right-radius: 80px;
+  padding-top: 90px;
+}
+
+.about-section h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-section p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.about-section .lets-connect {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  gap: 16px;
+}
+
+.about-section .get-resume-btn {
+  background: none;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 18px;
+  cursor: pointer;
+}
+
+.about-section .get-resume-btn:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.lang-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  padding: 44px 0;
+  color: #000;
+}
+
+.languages {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.languages > li {
+  background-color: #f7f7f9;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
```

Comment:
- [ ] Kindly make sure to add the padding on all sides of the language card in order to meet the provided Figma design.

   ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/228074111-50f8c898-e7ce-4e00-bb50-96556fbcd8c1.png)


   ### FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/228074047-64a37073-3b4f-4752-a724-cf93fac6a0ea.png)